tests/Integration/Behaviour/Features/Context/ProductFeatureContext.php

```diff
@@ -496,15 +496,15 @@ public function checkProductWithNameExists($productName)
         $this->checkFixtureExists($this->products, 'Product', $productName);
     }
 
-    /* SPECIFIC PRICE */
+    /* CUSTOM PRICE */
 
     /**
      * @Given /^product "(.+)" has a specific price named "(.+)" with an amount discount of (\d+\.\d+)$/
      */
     public function productWithNameHasASpecificPriceWithAmountDiscount($productName, $specificPriceName, $specificPriceDiscount)
     {
         if (isset($this->specificPrices[$productName][$specificPriceName])) {
-            throw new \Exception('Product named "' . $productName . '" has already a specific price named "' . $specificPriceName . '"');
+            throw new \Exception(sprintf('Product named %s has already a specific price named %s', $productName, $specificPriceName));
```

Comment:
```suggestion
            throw new Exception(sprintf('Product named %s has already a specific price named %s', $productName, $specificPriceName));
```

No need for `\` here everywhere it seems it's only integrated with `use` statement